fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -389,6 +420,55 @@ public static ConstantOperator divideDecimal(ConstantOperator first, ConstantOpe
         return createDecimalConstant(first.getDecimal().divide(second.getDecimal()));
     }
 
+    @FEFunction(name = "int_divide", argTypes = {"TINYINT", "TINYINT"}, returnType = "TINYINT")
```

Comment:
Plz add test to cover this functions

Reply:
OK. Already move those functions add in another PR #4807